apm-dist/release-docs/LICENSE

```diff
@@ -255,6 +256,7 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
     Apache: commons-configuration 1.8: https://github.com/apache/commons-configuration, Apache 2.0
     Apache: commons-io 2.4: https://github.com/apache/commons-io, Apache 2.0
     Apache: tomcat 8.5.27: https://github.com/apache/tomcat/tree/trunk, Apache 2.0
+    Apache: groovy 2.4.5-indy: https://github.com/apache/groovy, Apache 2.0
```

Comment:
Groovy has a NOTICE file. Please change NOTICE in same folder. And follow the file structure. Ref here: https://github.com/apache/groovy/blob/master/NOTICE